File: src/leetcode/344. Reverse String/solution.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        l = len(s)
        i = 0
        j = l - 1
        while i < j:
            s[i], s[j] = s[j], s[i]
            i += 1
            j -= 1

File: src/leetcode/344. Reverse String/solution.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int i = 0;
        int n = s.size() - 1;
        while(i < n) {
            char tmp = s[i];
            s[i] = s[n];
            s[n] = tmp;

            i++;
            n--;
        }
    }
};
